### DISCUSSION TITLE
Change mute audio color from purple to something else/ tab bar button colors

### DISCUSSION THREAD

**Discussion by rohitsaha29** [Alex313031/thorium#165] [Q&A]
Hey, just got thorium and its been phenomenal so far, but I was wondering if there is anyway for me to change the button and icon cars in the tab bar and the purple mute button to say all white? Please let me know

Thank you in advance!

**Comment by gz83** [comment]
The current purple icon is hard-coded into the code and is not yet open for users to change, the experimental flags for color change may be added subsequently

@rohitsaha29

**Comment by gz83** [comment]
The new version no longer seems to have this issue.
